hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1901,4 +1901,24 @@ possible configurations would overwhelm and obscure the important.
       automatically deleted until it is manually deleted
     </description>
   </property>
+  <property>
+    <name>hbase.master.regions.recovery.interval</name>
+    <value>1200000</value>
+    <description>
+      Regions Recovery Chore interval in milliseconds.
+      This chore keeps running at this interval to
+      find all regions with high store ref count and
+      reopens them.
+    </description>
+  </property>
+  <property>
+    <name>hbase.regions.recovery.store.file.count</name>
+    <value>128</value>
```

Comment:
May be a default of 1000 or so better.
Ya safe side leave META region from this as of now?